PermissionsScraper/PermissionsScraper/Triggers/DescriptionsScraper.cs

```diff
@@ -160,11 +171,16 @@ public static void Run([TimerTrigger("0 0 9 * * 1-5")]TimerInfo myTimer, ILogger
             /* Fetch permissions defined in the second level dictionary(ies),
              * e.g. appRoles, oauth2PermissionScopes --> 2nd level dictionary keys
              */
-            foreach (var scopeName in config.ScopesNames)
+            foreach (string scopeName in config.ScopesNames)
             {
                 // Retrieve all scopes descriptions for a given 2nd level dictionary retrieved from the Service Principal
                 var scopeDescriptions = spValue.First.Value<JArray>(scopeName)?.ToObject<List<Dictionary<string, object>>>();
 
+                if (scopeDescriptions == null)
+                {
+                    break;
```

Comment:
`continue`?

Because one value for a scopeName could be null, but maybe not the next.